examples/17_yaml.rs

```diff
@@ -42,4 +43,11 @@ fn main() {
     } else {
         println!("--mode <MODE> wasn't used...");
     }
-}
\ No newline at end of file
+}
+
+#[cfg(not(feature = "yaml"))]
+fn main() {
+    // As stated above, if clap is not compiled YAML feature, it is disabled.
```

Comment:
> As stated above, if clap is not compiled YAML feature, it is disabled

Should be "As stated above, if clap is not compiled **with the** YAML feature, it is disabled"